manifests/image-references

```diff
@@ -6,6 +6,10 @@ spec:
     from:
       kind: DockerImage
       name: registry.ci.openshift.org/openshift:cluster-capi-operator
+  - name: cluster-api
+    from:
+      kind: DockerImage
+      name: registry.ci.openshift.org/openshift:cluster-api
```

Comment:
This image is definitely in payload right?

Reply:
Looks like no, I thought it is at this moment

Reply:
Now I remember, Clayton asked to call it `cluster-capi-controllers`. If `images` task passes in CI then it's fine.